zproject/settings.py

```diff
@@ -313,25 +313,18 @@ def get_dirs(self) -> List[str]:
 
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
 
-# Compress large values being stored in memcached; this is important
-# for at least the realm_users cache.
-PYLIBMC_MIN_COMPRESS_LEN = 100 * 1024
-PYLIBMC_COMPRESS_LEVEL = 1
-
 MEMCACHED_PASSWORD = get_secret("memcached_password")
 
 CACHES = {
     'default': {
-        'BACKEND': 'django_pylibmc.memcached.PyLibMCCache',
+        'BACKEND': 'django_bmemcached.memcached.BMemcached',
         'LOCATION': MEMCACHED_LOCATION,
-        'TIMEOUT': 3600,
-        'BINARY': True,
-        'USERNAME': MEMCACHED_USERNAME,
-        'PASSWORD': MEMCACHED_PASSWORD,
         'OPTIONS': {
-            'tcp_nodelay': True,
-            'retry_timeout': 1,
```

Comment:
bmemcached doesn’t have an option for `TCP_NODELAY`, although I’m not sure that matters when every send is followed by a receive. It looks like it has a 5 second retry timeout.

Reply:
It looks like most advice for pylibmc and a few similar memcached libraries recommends `tcp_nodelay`.  Maybe worth asking the bmemcached authors if it's intentional that they don't provide the option?  I don't see a clear resource on whether it's useful or just cargo-culting.